src/loaders/configure/configure.js

```diff
@@ -55,6 +57,8 @@ export function configure (agent, opts = {}, loaderType, forceDrain) {
   runtime.ptid = agent.agentIdentifier
   setRuntime(agent.agentIdentifier, runtime)
 
+  agent.ee = ee.get(agent.agentIdentifier)
```

Comment:
the `ee` set up was moved to the configure call instead of the agent base, to solve the issue comment above for the micro agent